scripts/exglobal_atmos_analysis.sh

```diff
@@ -238,62 +238,62 @@ USE_CORRELATED_OBERRS=${USE_CORRELATED_OBERRS:-"YES"}
+      LATB_ENKF=${LATB_ENKF:-$(${NCLEN} ${ATMGES_ENSMEAN} grid_yt)} # get LATB_ENFK
+      LEVS_ENKF=${LEVS_ENKF:-$(${NCLEN} ${ATMGES_ENSMEAN} pfull)} # get LATB_ENFK
       JCAP_ENKF=${JCAP_ENKF:--9999} # again, no jcap in the netcdf files
    else
-      LONB_ENKF=${LONB_ENKF:-$($NEMSIOGET $ATMGES_ENSMEAN dimx | grep -i "dimx" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LONB_ENKF
-      LATB_ENKF=${LATB_ENKF:-$($NEMSIOGET $ATMGES_ENSMEAN dimy | grep -i "dimy" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LATB_ENKF
-      LEVS_ENKF=${LEVS_ENKF:-$($NEMSIOGET $ATMGES_ENSMEAN dimz | grep -i "dimz" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LEVS_ENKF
-      JCAP_ENKF=${JCAP_ENKF:-$($NEMSIOGET $ATMGES_ENSMEAN jcap | grep -i "jcap" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get JCAP_ENKF
+      LONB_ENKF=${LONB_ENKF:-$(${NEMSIOGET} ${ATMGES_ENSMEAN} dimx | grep -i "dimx" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LONB_ENKF
+      LATB_ENKF=${LATB_ENKF:-$(${NEMSIOGET} ${ATMGES_ENSMEAN} dimy | grep -i "dimy" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LATB_ENKF
+      LEVS_ENKF=${LEVS_ENKF:-$(${NEMSIOGET} ${ATMGES_ENSMEAN} dimz | grep -i "dimz" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get LEVS_ENKF
+      JCAP_ENKF=${JCAP_ENKF:-$(${NEMSIOGET} ${ATMGES_ENSMEAN} jcap | grep -i "jcap" | awk -F"= " '{print $2}' | awk -F" " '{print $1}')}  # 'get JCAP_ENKF
    fi
-   NLON_ENKF=${NLON_ENKF:-$LONB_ENKF}
-   NLAT_ENKF=${NLAT_ENKF:-$(($LATB_ENKF+2))}
-   [ $JCAP_ENKF -eq -9999 -a $LATB_ENKF -ne -9999 ] && JCAP_ENKF=$((LATB_ENKF-2))
-   [ $LONB_ENKF -eq -9999 -o $LATB_ENKF -eq -9999 -o $LEVS_ENKF -eq -9999 -o $JCAP_ENKF -eq -9999 ] && exit -9999
+   NLON_ENKF=${NLON_ENKF:-${LONB_ENKF}}
+   NLAT_ENKF=${NLAT_ENKF:-$((${LATB_ENKF}+2))}
+   [ ${JCAP_ENKF} -eq -9999 -a ${LATB_ENKF} -ne -9999 ] && JCAP_ENKF=$((LATB_ENKF-2))
+   [ ${LONB_ENKF} -eq -9999 -o ${LATB_ENKF} -eq -9999 -o ${LEVS_ENKF} -eq -9999 -o ${JCAP_ENKF} -eq -9999 ] && exit -9999
 else
    LONB_ENKF=0 # just for if statement later
 fi
 
 # Get dimension information based on CASE
-res=$(echo $CASE | cut -c2-)
+res=$(echo ${CASE} | cut -c2-)
 JCAP_CASE=$((res*2-2))
 LATB_CASE=$((res*2))
 LONB_CASE=$((res*4))
 
 # Set analysis resolution information
-if [ $DOHYBVAR = "YES" ]; then
-   JCAP_A=${JCAP_A:-${JCAP_ENKF:-$JCAP}}
-   LONA=${LONA:-${LONB_ENKF:-$LONB}}
-   LATA=${LATA:-${LATB_ENKF:-$LATB}}
+if [ ${DOHYBVAR} = "YES" ]; then
+   JCAP_A=${JCAP_A:-${JCAP_ENKF:-${JCAP}}}
+   LONA=${LONA:-${LONB_ENKF:-${LONB}}}
+   LATA=${LATA:-${LATB_ENKF:-${LATB}}}
 else
-   JCAP_A=${JCAP_A:-$JCAP}
-   LONA=${LONA:-$LONB}
-   LATA=${LATA:-$LATB}
+   JCAP_A=${JCAP_A:-${JCAP}}
+   LONA=${LONA:-${LONB}}
+   LATA=${LATA:-${LATB}}
 fi
-NLON_A=${NLON_A:-$LONA}
-NLAT_A=${NLAT_A:-$(($LATA+2))}
+NLON_A=${NLON_A:-${LONA}}
+NLAT_A=${NLAT_A:-$((${LATA}+2))}
```

Comment:
`${ .... }` shouldn't be necessary in a mathematical expression. Use `$((LATA+2))`